### PR TITLE
add: navbar and sidebar to documentation

### PR DESCRIPTION
**Pull request by ParthParolekar**
Added navbar and sidebar to documentation and made it responsive.

### CODE REVIEW

**Review by ApurvaSawant11 — APPROVED** [review]
Looks nice. You can add media queries for mobile as well.

**Comment by ParthParolekar** [comment]
> Looks nice. You can add media queries for mobile as well.

Thank you! :) I used mobile-first approach and added media queries for larger screens.

File: Documentation/Docs-CSS/navbar.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+}
+
+.logo {
+  margin-left: 1rem;
+  cursor: pointer;
+}
+
+.hamburger {
+  font-size: 1.5rem;
+  cursor: pointer;
```
Comment (nayyyhaa) [inline Documentation/Docs-CSS/navbar.css]: I see `cursor: pointer` being used multiple times. You could instead create a class fo that & use it wherever required.

**Review by nayyyhaa — COMMENTED** [review]
Nice work! :)

File: Documentation/Docs-CSS/navbar.css (hunk 1 above)
Comment (ParthParolekar) [reply Documentation/Docs-CSS/navbar.css]: Will do it. Thanks for the suggestion!

**Pull request merged**